Userland/Utilities/ping.cpp

```diff
@@ -180,6 +183,14 @@ int main(int argc, char** argv)
                 return 1;
             }
 
+            char internet_header_length = -1;
```

Comment:
It seems like it's unnecessary to initialize this variable only on the next line.

Also, don't we have `i8` as a nicer type alias?